ui/components/header-footer.js

```diff
@@ -91,5 +91,6 @@ export default function HeaderFooter({ children }) {
 }
 
 HeaderFooter.propTypes = {
+  showSearch: PropTypes.bool,
```

Comment:
Heads up, we might get a linting error here (once linting is part of CI again) due to not having a default.

Reply:
Copy!